src/shm_manager.cc

```diff
@@ -76,7 +76,7 @@ SharedMemoryManager::SharedMemoryManager(
          "' to requested size (" + std::to_string(shm_size) +
          " bytes). If you are running Triton inside docker, use '--shm-size' "
          "flag to control the shared memory region size. Each Python backend "
-         "model instance requires at least 64MBs of shared memory. Error: " +
+         "model instance requires at least 1 MB of shared memory. Error: " +
```

Comment:
nit if you feel like changing this:

We can just define a constant and use it everywhere so we don't have to remember to fix each spot:
```cpp
// constants.h
constexpr uint64_t DEFAULT_SHM_SIZE = 1*1024*1024; // 1 MB
constexpr uint64_t DEFAULT_SHM_GROWTH_SIZE = 1*1024*1024; // 1 MB

...
// source
backend_state->shm_default_byte_size = DEFAULT_SHM_SIZE;
...
        if (backend_state->shm_default_byte_size < DEFAULT_SHM_SIZE) {
...
error_message = "... at least " + std::to_string(DEFAULT_SHM_SIZE) + " of shared memory..."
```

Reply:
It is already only in two places so I think it should be fine.